src/carbEntryScreen.js

```diff
@@ -8,21 +8,94 @@ class CarbEntryScreen {
     async Open() {
         try {
             //assume we are starting from the open screen
-            await match.accessible.Button(CarbsLabel.AddMeal).tap();
+            await match.accessible.Button(text.carbEntryScreen.AddMeal).tap();
         } catch (err) { } //catch and continue
     }
     /**
      * @example await carbs.Cancel();
      */
     async Cancel() {
-        await match.accessible.ButtonBarButton(Label.Cancel).tap();
+        await this.CancelHeaderButton().tap();
+    }
+    CancelHeaderButton() {
+        return match.accessible.ButtonBarButton(text.general.Cancel);
+    }
+    AddCarbEntryHeader() {
+        return match.accessible.Header(text.carbEntryScreen.AddCarbEntry);
+    }
+    AmountConsumedLabel() {
+        return match.accessible.Label(text.carbEntryScreen.AmountConsumed);
+    }
+    DateLabel() {
+        return match.accessible.Label(text.carbEntryScreen.Date);
+    }
+    FoodTypeLabel() {
+        return match.accessible.Label(text.carbEntryScreen.FoodType);
+    }
+    AbsorptionTimeLabel() {
+        return match.accessible.Label(text.carbEntryScreen.AbsorptionTime);
+    }
+    /**
```

Comment:
[nit] I would drop most of these comments like this. I don't see them adding much useful, and, at least in this case, it is wrong. ;)

Reply:
I'd only add comments if it adds information that isn't already provided by the name of the function. Perhaps if there is a caveat or special considerations. That sort of thing.